packages/angular/src/directives/navigation/ion-tabs.ts

```diff
@@ -11,11 +11,13 @@ import { IonRouterOutlet } from './ion-router-outlet';
     <ng-content select="[slot=top]"></ng-content>
     <div class="tabs-inner" #tabsInner>
       <ion-router-outlet
+        *ngIf="tabs?.length === 0"
```

Comment:
Can `tabs` be undefined? The type signature implies not.

Reply:
After reviewing it further, it seems that there isn't a moment that it would be undefined. No issues were found when removing the optional chain. https://github.com/ionic-team/ionic-framework/pull/29786/commits/f6132ca59a65c55f285ed0aad7f5e45a8e1cdc91